datafusion/physical-plan/src/sorts/partial_sort.rs

```diff
@@ -390,6 +390,7 @@ impl PartialSortStream {
     fn sort_in_mem_batches(self: &mut Pin<&mut Self>) -> Result<RecordBatch> {
         let input_batch = concat_batches(&self.schema(), &self.in_mem_batches)?;
         self.in_mem_batches.clear();
+        println!("input_batch: {:?}", input_batch);
```

Comment:
Looks like some debug logging that needs to be removed?

Reply:
Oops, yea.